### ISSUE TITLE
Custom Enchantments from my own mod not working. mod-interaction

### ISSUE THREAD

**Issue by Zxaleb27** [Shadows-of-Fire/Apotheosis#952]
I have a mod that creates an enchant that goes from level 1-3, and having apotheosis enabled alongside it makes the enchant only go to level 1

**Comment by Shadows-of-Fire** [comment]
What do you mean goes from level 1-3, is the max level 1 or 3?

Apotheosis does not reduce the default max levels of any enchantments, unless for some reason your default enchanting power functions were really screwed up (to the point where only level 1 was ever available in an enchanting table).

Could you show the source code of your enchantment?

**Comment by Zxaleb27** [comment]
The enchantment was made using MCreator, so it should have default code, but i'll send anyways. And it goes from level 1 to 3. It may also be a problem with Apotheosis AND Ensorcellation, so I'll test that. The actual function of the enchantment doesn't matter, since it's handled by a separate procedure, and still works fine. The way the enchantment procedure works is by doing a bunch of checks if you actually have it, are in the air, and have not air jumped more times than the enchant level, then overrides your motion vector, adds one to your jumps count and then does some funky particle/sound stuff, if you're interested.
![image_2023-07-18_003201845](https://github.com/Shadows-of-Fire/Apotheosis/assets/47232352/0943410d-3117-465d-a6a0-b541d1e010fe)
![image](https://github.com/Shadows-of-Fire/Apotheosis/assets/47232352/eb074d1e-1ead-4c57-ad9c-6826bacd2646)
![image](https://github.com/Shadows-of-Fire/Apotheosis/assets/47232352/ff650ffb-c0e2-4108-b1d1-cb8f34da3467)

(After enabling only my mod and Apotheosis:) The issue is with Apotheosis, not Apo + Ensorcellation. Checking only mine + Placebo fixes the issue.

**Comment by Shadows-of-Fire** [comment]
Reset your apotheosis configs, then install your mod - if the issue persists, please pull your config file. The behavior you're experiencing _shouldn't_ be happening by default, unless you booted the game with your enchantment and the max level got locked to 1 in the config.

**Comment by Zxaleb27** [comment]
I checked the config for my enchant. Found it, and found the culprit. I'm not entirely sure **why** but the max level was 1, but it says it was originally 3.
![image](https://github.com/Shadows-of-Fire/Apotheosis/assets/47232352/05d845da-3525-4c53-8ee6-bd7ce547a82f)
This is likely due to the fact it only went to 1 in my 1.0 build, but went to 3 in my 1.1 build. Not the fault of Apotheosis, but something maybe worth looking into a check/easy fix for. Perhaps having the default as a separate value that **always** updates to the mod's default, then having a changeable value and resetting that one if it hasn't been tampered with? Unneccesary, but would fix a small oversight.
